modules/core-services/endpoints/application.go

```diff
@@ -527,17 +527,30 @@ func (e *Endpoints) listApplications(ctx context.Context, r *http.Request, isMin
 	unpinedAppDTOs := make([]apistructs.ApplicationDTO, 0, len(applications))
 
 	for i := range applications {
+		var projectName string
+		var projectDisplayName string
+		project, err := e.project.GetModelProject(applications[i].ProjectID)
+		if err == nil {
+			projectName = project.Name
+			projectDisplayName = project.DisplayName
+		} else {
+			logrus.Error(err)
+		}
```

Comment:
it's better query projects by one invoke like:

projectIds := []int
for i := range apps {
  append(projectIds, .)
}
getProjectByIds(projectIds)

Reply:
done